# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nethack/unixres.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./sys/unix/unixres.c	2003-02-23 15:43:39.000000000 +0100
+++ ./sys/unix/unixres.c	2003-06-08 13:45:58.000000000 +0200
@@ -24,6 +24,7 @@
 #endif
 
 /* requires dynamic linking with libc */
+#define __USE_GNU
 #include <dlfcn.h>
 
 static int
